docs/plugins/Store/oras.md

```diff
@@ -1,10 +1,97 @@
+ORAS is a built-in `Store` plugin. It leverages the [ORAS project](https://github.com/oras-project) to authenticate, discover, list, and download artifacts from OCI compliant registries.
+The ORAS store leverages the `oras-go` [library](https://github.com/oras-project/oras-go).
+
+ORAS store is the default and currently ONLY supported `Store` implementation.
+
+Here's an example `oras` referrer store CRD spec for a local development registy (insecure + tls-check disabled) that caches registry content.
+
+```yml
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Store
+metadata:
+  name: store-oras
+spec:
+  name: oras
+  parameters: 
+    cacheEnabled: true
+    ttl: 10
+    useHttp: true
+```
+
+## Configuration
+
+### Kubernetes
+
+```yml
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Store
+metadata:
+  name: store-oras
+spec:
+  name: oras
+  parameters: 
+    cacheEnabled: # defaults to true
+    ttl: # TTL in seconds for ORAS cache. Default is 10
+    useHttp: # Local testing ONLY. Disables TLS checks uses HTTP. Default is false.
+    cosignEnabled: # enables discovery of cosign artifacts from registry. Default is false.
+    localCachePath: # absolute file path to an existing/new ORAS OCI local store
+    authProvider:
+        name: # name of the auth provider type
+        # auth provider specific fields here
+```
+
+| Name           | Required | Description                                                                                                                                                                                                        | Default Value               |
+| -------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------- |
+| cosignEnabled  | no       | This must be `true` if cosign verifier is enabled. Read more about cosign verifier [here](https://github.com/deislabs/ratify/blob/main/plugins/verifier/cosign/README.md).                                         | `false`                     |
+| authProvider   | no       | This is only required if pulling from a private repository. For all supported auth mode, please review [oras-auth-provider](https://github.com/deislabs/ratify/blob/main/docs/reference/oras-auth-provider.md) doc | dockerAuth                  |
+| cacheEnabled   | no       | Oras cache, cache for all referrers for a subject. Note: global cache must be enabled first                                                                                                                        | `false`                     |
```

Comment:
curious why defaut is false

Reply:
good catch. these are not up to date. let me fix this

Reply:
> good catch. these are not up to date. let me fix this

Actually scratch that. It is correct in it's current form. All unspecified fields (ex: user doesn't add `cacheEnabled` in the config) will default to `false` value. This is due to behavior of `json.Unmarshal`. To set custom defaults per config, we wuold need to introduce that in ORAS store implementation. This is also why some projects never use the raw `bool` type directly. They use a pointer `*bool`. This allows for three states (nil, true, false) where the unspecified has it's own `nil` state.